torch/lib/c10d/reducer.hpp

```diff
@@ -421,6 +435,11 @@ class Reducer {
 
   bool static_graph_ = false;
 
+  // Map will not change after 1st iteration.
```

Comment:
nit: Would be good to add a line explaining what the maps are responsible for doing.